spec/integration/util/windows/security_spec.rb

```diff
@@ -476,6 +476,7 @@ def grant_everyone_full_access(path)
           it "should retrieve the user sid" do
             sid = nil
             user = Puppet::Util::Windows::ADSI::User.create("puppet#{rand(10000)}")
+            user.password = 'PUP123!'
```

Comment:
(facepalm)

Originally I was using random GUIDs, but then modified to make sure that we always have the 3 of 5 categories covered and just removed the GUID... totally spaced on length!